tasks/main.yml

```diff
@@ -161,8 +150,8 @@
     src: "{{ item.value.src_check_script }}"
     dest: "{{ item.value.dest_check_script | default(item.value.check_script) }}"
     mode: "0755"
-  loop: "{{ keepalived_sync_groups | dict2items | default('{}') }}"
-  when: item.value.src_check_script is defined
+  loop: "{{ keepalived_scripts | dict2items | default('{}') }}"
```

Comment:
keepalived_scripts | default({}) | dict2items.

Without this, keepalived_scripts is by default undefined. You can probably add keepalived_scripts: {} in defaults if you find this more clean...

Reply:
Or we can add `keepalived_scripts` to defaults/main.yml with some example so that it was more obvious that this variable is available for use?

Reply:
Kinda close to what I have said a bit earlier? :)
The documentation is not in defaults, but simply in README. I don't mind a duplication here.

Reply:
Side note: I indeed prefer to have an example in defaults/, and refer to it in README. :)

Reply:
i will update the readme to provide an example (that's already provided in /tests)